INFO 2018-08-18 16:38:51,292 basehttp 124 "GET / HTTP/1.1" 200 1714
WARNING 2018-08-18 16:38:51,349 base 152 Not Found: /favicon.ico
WARNING 2018-08-18 16:38:51,350 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 1972
WARNING 2018-08-18 16:40:37,434 base 152 Not Found: /
WARNING 2018-08-18 16:40:37,436 basehttp 124 "GET / HTTP/1.1" 404 2111
WARNING 2018-08-18 16:51:02,205 base 152 Not Found: /mobiles/13112345678/count/
WARNING 2018-08-18 16:51:02,212 base 152 Not Found: /mobiles/13112345678/count/
WARNING 2018-08-18 16:51:02,223 base 152 Not Found: /sms_code/13112345678/
WARNING 2018-08-18 16:51:02,224 basehttp 124 "GET /mobiles/13112345678/count/ HTTP/1.1" 404 2207
WARNING 2018-08-18 16:51:02,231 basehttp 124 "GET /mobiles/13112345678/count/ HTTP/1.1" 404 2207
WARNING 2018-08-18 16:51:02,232 basehttp 124 "GET /sms_code/13112345678/ HTTP/1.1" 404 2186
WARNING 2018-08-18 16:51:16,471 base 152 Not Found: /sms_code/13112345678/
WARNING 2018-08-18 16:51:16,472 basehttp 124 "GET /sms_code/13112345678/ HTTP/1.1" 404 2186
WARNING 2018-08-18 16:51:16,477 base 152 Not Found: /mobiles/13112345678/count/
WARNING 2018-08-18 16:51:16,478 basehttp 124 "GET /mobiles/13112345678/count/ HTTP/1.1" 404 2207
WARNING 2018-08-18 16:53:09,605 base 152 Not Found: /mobiles/13112345678/count/
WARNING 2018-08-18 16:53:09,607 basehttp 124 "GET /mobiles/13112345678/count/ HTTP/1.1" 404 2206
INFO 2018-08-18 16:53:09,625 basehttp 124 "GET /sms_code/13112345678/ HTTP/1.1" 200 16
WARNING 2018-08-18 20:34:13,807 base 152 Not Found: /mobiles/15222366545/count/
WARNING 2018-08-18 20:34:13,811 basehttp 124 "GET /mobiles/15222366545/count/ HTTP/1.1" 404 2206
WARNING 2018-08-18 20:38:15,388 base 152 Not Found: /mobiles/15222366545/count/
WARNING 2018-08-18 20:38:15,389 basehttp 124 "GET /mobiles/15222366545/count/ HTTP/1.1" 404 2206
WARNING 2018-08-18 20:38:15,502 base 152 Not Found: /mobiles/15222366545/count/
WARNING 2018-08-18 20:38:15,503 basehttp 124 "GET /mobiles/15222366545/count/ HTTP/1.1" 404 2206
INFO 2018-08-18 20:38:15,519 basehttp 124 "GET /sms_code/15222366545/ HTTP/1.1" 200 16
WARNING 2018-08-18 20:51:46,917 base 152 Not Found: /mobiles/15222366545/count/
INFO 2018-08-18 20:51:46,918 basehttp 124 "GET /sms_code/15222366545/ HTTP/1.1" 200 16
WARNING 2018-08-18 20:51:46,924 basehttp 124 "GET /mobiles/15222366545/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:13:45,756 base 152 Not Found: /mobiles/15444669988/count/
WARNING 2018-08-19 08:13:45,758 basehttp 124 "GET /mobiles/15444669988/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:13:45,768 base 152 Not Found: /mobiles/15444669988/count/
WARNING 2018-08-19 08:13:45,771 basehttp 124 "GET /mobiles/15444669988/count/ HTTP/1.1" 404 2206
INFO 2018-08-19 08:13:45,785 basehttp 124 "GET /sms_code/15444669988/ HTTP/1.1" 200 16
WARNING 2018-08-19 08:13:50,677 base 152 Not Found: /mobiles/15444669988/count/
WARNING 2018-08-19 08:13:50,679 basehttp 124 "GET /mobiles/15444669988/count/ HTTP/1.1" 404 2206
ERROR 2018-08-19 08:13:50,680 exception 135 Internal Server Error: /sms_code/15444669988/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/workspace/Web/Django/meiduo/meiduo_api/apps/verifications/views.py", line 17, in get
    raise serializers.ValidationError("请勿重复发送")
rest_framework.exceptions.ValidationError: [ErrorDetail(string='请勿重复发送', code='invalid')]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 183, in import_from_string
    module = import_module(module_path)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'meiduo_mall'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 437, in handle_exception
    exception_handler = self.get_exception_handler()
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 288, in get_exception_handler
    return self.settings.EXCEPTION_HANDLER
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 227, in __getattr__
    val = perform_import(val, attr)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 170, in perform_import
    return import_from_string(val, setting_name)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 187, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'meiduo_mall.utils.exceptions.exception_handler' for API setting 'EXCEPTION_HANDLER'. ImportError: No module named 'meiduo_mall'.
ERROR 2018-08-19 08:13:50,762 basehttp 124 "GET /sms_code/15444669988/ HTTP/1.1" 500 120825
WARNING 2018-08-19 08:14:52,109 base 152 Not Found: /mobiles/13455669988/count/
WARNING 2018-08-19 08:14:52,109 basehttp 124 "GET /mobiles/13455669988/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:14:55,713 base 152 Not Found: /mobiles/13455669988/count/
WARNING 2018-08-19 08:14:55,714 basehttp 124 "GET /mobiles/13455669988/count/ HTTP/1.1" 404 2206
INFO 2018-08-19 08:14:55,812 basehttp 124 "GET /sms_code/13455669988/ HTTP/1.1" 200 16
WARNING 2018-08-19 08:14:55,821 base 152 Not Found: /mobiles/13455669988/count/
WARNING 2018-08-19 08:14:55,821 basehttp 124 "GET /mobiles/13455669988/count/ HTTP/1.1" 404 2206
ERROR 2018-08-19 08:15:32,079 exception 135 Internal Server Error: /sms_code/13455669988/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/workspace/Web/Django/meiduo/meiduo_api/apps/verifications/views.py", line 17, in get
    raise serializers.ValidationError("请勿重复发送")
rest_framework.exceptions.ValidationError: [ErrorDetail(string='请勿重复发送', code='invalid')]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 183, in import_from_string
    module = import_module(module_path)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'meiduo_mall'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 437, in handle_exception
    exception_handler = self.get_exception_handler()
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 288, in get_exception_handler
    return self.settings.EXCEPTION_HANDLER
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 227, in __getattr__
    val = perform_import(val, attr)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 170, in perform_import
    return import_from_string(val, setting_name)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 187, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'meiduo_mall.utils.exceptions.exception_handler' for API setting 'EXCEPTION_HANDLER'. ImportError: No module named 'meiduo_mall'.
WARNING 2018-08-19 08:15:32,103 base 152 Not Found: /mobiles/13455669988/count/
WARNING 2018-08-19 08:15:32,112 basehttp 124 "GET /mobiles/13455669988/count/ HTTP/1.1" 404 2206
ERROR 2018-08-19 08:15:32,181 basehttp 124 "GET /sms_code/13455669988/ HTTP/1.1" 500 121086
WARNING 2018-08-19 08:17:23,956 base 152 Not Found: /mobiles/13455669988/count/
WARNING 2018-08-19 08:17:23,964 basehttp 124 "GET /mobiles/13455669988/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:17:24,972 base 152 Not Found: /mobiles/13455669988/count/
WARNING 2018-08-19 08:17:24,973 basehttp 124 "GET /mobiles/13455669988/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:17:25,056 base 152 Not Found: /mobiles/13455669988/count/
WARNING 2018-08-19 08:17:25,059 basehttp 124 "GET /mobiles/13455669988/count/ HTTP/1.1" 404 2206
INFO 2018-08-19 08:17:25,072 basehttp 124 "GET /sms_code/13455669988/ HTTP/1.1" 200 16
WARNING 2018-08-19 08:18:10,632 base 152 Not Found: /mobiles/15222641027/count/
WARNING 2018-08-19 08:18:10,633 basehttp 124 "GET /mobiles/15222641027/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:18:56,012 base 152 Not Found: /mobiles/15222641027/count/
WARNING 2018-08-19 08:18:56,013 basehttp 124 "GET /mobiles/15222641027/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:19:01,329 base 152 Not Found: /mobiles/15222641027/count/
WARNING 2018-08-19 08:19:01,329 basehttp 124 "GET /mobiles/15222641027/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:19:01,432 base 152 Not Found: /mobiles/15222641027/count/
WARNING 2018-08-19 08:19:01,434 basehttp 124 "GET /mobiles/15222641027/count/ HTTP/1.1" 404 2206
INFO 2018-08-19 08:19:01,434 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
WARNING 2018-08-19 08:19:48,106 base 152 Not Found: /mobiles/13565656464/count/
WARNING 2018-08-19 08:19:48,106 basehttp 124 "GET /mobiles/13565656464/count/ HTTP/1.1" 404 2206
INFO 2018-08-19 08:19:48,180 basehttp 124 "GET /sms_code/13565656464/ HTTP/1.1" 200 16
WARNING 2018-08-19 08:19:48,195 base 152 Not Found: /mobiles/13565656464/count/
WARNING 2018-08-19 08:19:48,196 basehttp 124 "GET /mobiles/13565656464/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:20:06,564 base 152 Not Found: /mobiles/13563636262/count/
WARNING 2018-08-19 08:20:06,565 basehttp 124 "GET /mobiles/13563636262/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:20:06,680 base 152 Not Found: /mobiles/13563636262/count/
WARNING 2018-08-19 08:20:06,681 basehttp 124 "GET /mobiles/13563636262/count/ HTTP/1.1" 404 2206
INFO 2018-08-19 08:20:06,691 basehttp 124 "GET /sms_code/13563636262/ HTTP/1.1" 200 16
WARNING 2018-08-19 08:20:30,805 base 152 Not Found: /mobiles/13563636263/count/
WARNING 2018-08-19 08:20:30,806 basehttp 124 "GET /mobiles/13563636263/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:20:31,865 base 152 Not Found: /mobiles/13563636263/count/
WARNING 2018-08-19 08:20:31,865 basehttp 124 "GET /mobiles/13563636263/count/ HTTP/1.1" 404 2206
INFO 2018-08-19 08:20:31,945 basehttp 124 "GET /sms_code/13563636263/ HTTP/1.1" 200 16
WARNING 2018-08-19 08:20:31,949 base 152 Not Found: /mobiles/13563636263/count/
WARNING 2018-08-19 08:20:31,949 basehttp 124 "GET /mobiles/13563636263/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:20:46,175 base 152 Not Found: /mobiles/13563636267/count/
WARNING 2018-08-19 08:20:46,177 basehttp 124 "GET /mobiles/13563636267/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:21:04,150 base 152 Not Found: /mobiles/13623231212/count/
WARNING 2018-08-19 08:21:04,151 basehttp 124 "GET /mobiles/13623231212/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 08:21:04,247 base 152 Not Found: /mobiles/13623231212/count/
WARNING 2018-08-19 08:21:04,255 basehttp 124 "GET /mobiles/13623231212/count/ HTTP/1.1" 404 2206
INFO 2018-08-19 08:21:04,256 basehttp 124 "GET /sms_code/13623231212/ HTTP/1.1" 200 16
WARNING 2018-08-19 15:57:11,934 base 152 Not Found: /mobiles/17633666633/count/
WARNING 2018-08-19 15:57:11,935 base 152 Not Found: /mobiles/17633666633/count/
WARNING 2018-08-19 15:57:11,939 basehttp 124 "GET /mobiles/17633666633/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 15:57:11,946 basehttp 124 "GET /mobiles/17633666633/count/ HTTP/1.1" 404 2206
INFO 2018-08-19 15:57:11,979 basehttp 124 "GET /sms_code/17633666633/ HTTP/1.1" 200 16
WARNING 2018-08-19 16:00:39,408 base 152 Not Found: /mobiles/17633666633/count/
WARNING 2018-08-19 16:00:39,411 basehttp 124 "GET /mobiles/17633666633/count/ HTTP/1.1" 404 2206
INFO 2018-08-19 16:00:39,485 basehttp 124 "GET /sms_code/17633666633/ HTTP/1.1" 200 16
WARNING 2018-08-19 19:13:45,612 base 152 Not Found: /mobiles/17633666633/count/
WARNING 2018-08-19 19:13:45,616 basehttp 124 "GET /mobiles/17633666633/count/ HTTP/1.1" 404 2206
INFO 2018-08-19 19:13:45,735 basehttp 124 "GET /sms_code/17633666633/ HTTP/1.1" 200 16
WARNING 2018-08-19 19:15:28,103 base 152 Not Found: /usernames/asdfghjkllkjhg/count/
WARNING 2018-08-19 19:15:28,107 basehttp 124 "GET /usernames/asdfghjkllkjhg/count/ HTTP/1.1" 404 2221
WARNING 2018-08-19 19:17:39,991 base 152 Not Found: /usernames/asdfghjkllkjhg/count/
WARNING 2018-08-19 19:17:39,993 basehttp 124 "GET /usernames/asdfghjkllkjhg/count/ HTTP/1.1" 404 2221
WARNING 2018-08-19 19:17:40,910 base 152 Not Found: /usernames/asdfghjkllkjhg/count/
WARNING 2018-08-19 19:17:40,911 basehttp 124 "GET /usernames/asdfghjkllkjhg/count/ HTTP/1.1" 404 2221
WARNING 2018-08-19 19:20:07,550 base 152 Not Found: /usernames/qwe1212121211/count/
WARNING 2018-08-19 19:20:07,558 basehttp 124 "GET /usernames/qwe1212121211/count/ HTTP/1.1" 404 2218
WARNING 2018-08-19 19:20:14,650 base 152 Not Found: /mobiles/13122332233/count/
WARNING 2018-08-19 19:20:14,656 basehttp 124 "GET /mobiles/13122332233/count/ HTTP/1.1" 404 2206
WARNING 2018-08-19 19:21:07,703 base 152 Not Found: /usernames/qwe1212121211/count/
WARNING 2018-08-19 19:21:07,703 basehttp 124 "GET /usernames/qwe1212121211/count/ HTTP/1.1" 404 2218
WARNING 2018-08-19 19:21:11,081 base 152 Not Found: /mobiles/13122332233/count/
WARNING 2018-08-19 19:21:11,082 basehttp 124 "GET /mobiles/13122332233/count/ HTTP/1.1" 404 2206
INFO 2018-08-19 19:22:03,030 basehttp 124 "GET /usernames/qwe1212121211/count/ HTTP/1.1" 200 38
INFO 2018-08-19 19:22:11,222 basehttp 124 "GET /mobiles/13122332233/count/ HTTP/1.1" 200 34
INFO 2018-08-19 19:37:22,827 basehttp 124 "GET /usernames/qwe1212121211/count/ HTTP/1.1" 200 38
INFO 2018-08-19 19:44:47,992 basehttp 124 "GET /usernames/qwe1212121211/count/ HTTP/1.1" 200 38
INFO 2018-08-19 20:07:28,933 basehttp 124 "GET /usernames/mrping000000/count/ HTTP/1.1" 200 37
INFO 2018-08-19 20:07:44,878 basehttp 124 "GET /mobiles/13122332233/count/ HTTP/1.1" 200 34
INFO 2018-08-19 20:07:45,388 basehttp 124 "GET /mobiles/13122332233/count/ HTTP/1.1" 200 34
INFO 2018-08-19 20:07:45,465 basehttp 124 "GET /sms_code/13122332233/ HTTP/1.1" 200 16
INFO 2018-08-19 20:08:07,301 basehttp 124 "GET /usernames/mrping000000/count/ HTTP/1.1" 200 37
INFO 2018-08-19 20:08:07,307 basehttp 124 "OPTIONS /users/ HTTP/1.1" 200 0
WARNING 2018-08-19 20:08:07,315 base 152 Not Found: /users/
INFO 2018-08-19 20:08:07,317 basehttp 124 "GET /mobiles/13122332233/count/ HTTP/1.1" 200 34
WARNING 2018-08-19 20:08:07,325 basehttp 124 "POST /users/ HTTP/1.1" 404 2536
INFO 2018-08-19 20:08:50,820 basehttp 124 "GET /mobiles/13122332233/count/ HTTP/1.1" 200 34
INFO 2018-08-19 20:08:50,881 basehttp 124 "GET /sms_code/13122332233/ HTTP/1.1" 200 16
INFO 2018-08-19 20:09:08,902 basehttp 124 "OPTIONS /users/ HTTP/1.1" 200 0
INFO 2018-08-19 20:09:08,911 basehttp 124 "GET /usernames/mrping000000/count/ HTTP/1.1" 200 37
INFO 2018-08-19 20:09:08,955 basehttp 124 "GET /mobiles/13122332233/count/ HTTP/1.1" 200 34
INFO 2018-08-19 20:09:09,049 basehttp 124 "POST /users/ HTTP/1.1" 201 57
INFO 2018-08-19 20:34:35,924 basehttp 124 "GET /usernames/mrping1111/count/ HTTP/1.1" 200 35
INFO 2018-08-19 20:34:53,295 basehttp 124 "GET /mobiles/13244556655/count/ HTTP/1.1" 200 34
INFO 2018-08-19 20:34:53,373 basehttp 124 "GET /mobiles/13244556655/count/ HTTP/1.1" 200 34
INFO 2018-08-19 20:34:53,432 basehttp 124 "GET /sms_code/13244556655/ HTTP/1.1" 200 16
INFO 2018-08-19 20:35:10,190 basehttp 124 "GET /usernames/mrping1111/count/ HTTP/1.1" 200 35
INFO 2018-08-19 20:35:10,202 basehttp 124 "OPTIONS /users/ HTTP/1.1" 200 0
INFO 2018-08-19 20:35:10,221 basehttp 124 "GET /mobiles/13244556655/count/ HTTP/1.1" 200 34
INFO 2018-08-19 20:35:10,326 basehttp 124 "POST /users/ HTTP/1.1" 201 55
INFO 2018-08-19 20:36:13,597 basehttp 124 "GET /usernames/mrping3333/count/ HTTP/1.1" 200 35
INFO 2018-08-19 20:36:23,164 basehttp 124 "GET /mobiles/15600999900/count/ HTTP/1.1" 200 34
INFO 2018-08-19 20:36:23,297 basehttp 124 "GET /mobiles/15600999900/count/ HTTP/1.1" 200 34
INFO 2018-08-19 20:36:23,340 basehttp 124 "GET /sms_code/15600999900/ HTTP/1.1" 200 16
INFO 2018-08-19 20:36:35,716 basehttp 124 "GET /usernames/mrping3333/count/ HTTP/1.1" 200 35
INFO 2018-08-19 20:36:35,733 basehttp 124 "OPTIONS /users/ HTTP/1.1" 200 0
INFO 2018-08-19 20:36:35,735 basehttp 124 "GET /mobiles/15600999900/count/ HTTP/1.1" 200 34
INFO 2018-08-19 20:36:35,871 basehttp 124 "POST /users/ HTTP/1.1" 201 55
INFO 2018-08-21 14:56:29,583 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:56:31,812 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:56:33,972 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:56:34,593 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:56:34,760 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:56:34,902 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:56:35,046 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:56:35,199 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:56:35,353 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:56:35,505 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:56:47,163 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:56:56,669 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:58:58,792 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 14:58:58,952 basehttp 124 "POST /authorizations/ HTTP/1.1" 200 180
WARNING 2018-08-21 14:59:12,673 base 152 Not Found: /
WARNING 2018-08-21 14:59:12,674 basehttp 124 "GET / HTTP/1.1" 404 2820
WARNING 2018-08-21 14:59:12,715 base 152 Not Found: /favicon.ico
WARNING 2018-08-21 14:59:12,718 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 2871
WARNING 2018-08-21 14:59:23,200 base 152 Not Found: /^authorizations/
WARNING 2018-08-21 14:59:23,201 basehttp 124 "GET /%5Eauthorizations/ HTTP/1.1" 404 2888
ERROR 2018-08-21 14:59:28,162 exception 135 Internal Server Error: /authorizations/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 157, in http_method_not_allowed
    raise exceptions.MethodNotAllowed(request.method)
rest_framework.exceptions.MethodNotAllowed: 方法 “GET” 不被允许。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 183, in import_from_string
    module = import_module(module_path)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'meiduo_mall'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 437, in handle_exception
    exception_handler = self.get_exception_handler()
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 288, in get_exception_handler
    return self.settings.EXCEPTION_HANDLER
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 227, in __getattr__
    val = perform_import(val, attr)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 170, in perform_import
    return import_from_string(val, setting_name)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 187, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'meiduo_mall.utils.exceptions.exception_handler' for API setting 'EXCEPTION_HANDLER'. ImportError: No module named 'meiduo_mall'.
ERROR 2018-08-21 14:59:28,282 basehttp 124 "GET /authorizations/ HTTP/1.1" 500 120736
INFO 2018-08-21 14:59:40,278 basehttp 124 "GET /authorizations HTTP/1.1" 301 0
ERROR 2018-08-21 14:59:40,284 exception 135 Internal Server Error: /authorizations/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 157, in http_method_not_allowed
    raise exceptions.MethodNotAllowed(request.method)
rest_framework.exceptions.MethodNotAllowed: 方法 “GET” 不被允许。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 183, in import_from_string
    module = import_module(module_path)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'meiduo_mall'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 437, in handle_exception
    exception_handler = self.get_exception_handler()
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 288, in get_exception_handler
    return self.settings.EXCEPTION_HANDLER
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 227, in __getattr__
    val = perform_import(val, attr)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 170, in perform_import
    return import_from_string(val, setting_name)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 187, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'meiduo_mall.utils.exceptions.exception_handler' for API setting 'EXCEPTION_HANDLER'. ImportError: No module named 'meiduo_mall'.
ERROR 2018-08-21 14:59:40,353 basehttp 124 "GET /authorizations/ HTTP/1.1" 500 120736
WARNING 2018-08-21 14:59:44,215 base 152 Not Found: /
WARNING 2018-08-21 14:59:44,216 basehttp 124 "GET / HTTP/1.1" 404 2820
ERROR 2018-08-21 14:59:51,932 exception 135 Internal Server Error: /authorizations/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 157, in http_method_not_allowed
    raise exceptions.MethodNotAllowed(request.method)
rest_framework.exceptions.MethodNotAllowed: 方法 “GET” 不被允许。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 183, in import_from_string
    module = import_module(module_path)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'meiduo_mall'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 437, in handle_exception
    exception_handler = self.get_exception_handler()
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 288, in get_exception_handler
    return self.settings.EXCEPTION_HANDLER
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 227, in __getattr__
    val = perform_import(val, attr)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 170, in perform_import
    return import_from_string(val, setting_name)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 187, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'meiduo_mall.utils.exceptions.exception_handler' for API setting 'EXCEPTION_HANDLER'. ImportError: No module named 'meiduo_mall'.
ERROR 2018-08-21 14:59:51,982 basehttp 124 "GET /authorizations/ HTTP/1.1" 500 120736
ERROR 2018-08-21 15:00:43,776 exception 135 Internal Server Error: /authorizations/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 157, in http_method_not_allowed
    raise exceptions.MethodNotAllowed(request.method)
rest_framework.exceptions.MethodNotAllowed: 方法 “GET” 不被允许。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 183, in import_from_string
    module = import_module(module_path)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'meiduo_api.utils'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 437, in handle_exception
    exception_handler = self.get_exception_handler()
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 288, in get_exception_handler
    return self.settings.EXCEPTION_HANDLER
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 227, in __getattr__
    val = perform_import(val, attr)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 170, in perform_import
    return import_from_string(val, setting_name)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 187, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'meiduo_api.utils.exceptions.exception_handler' for API setting 'EXCEPTION_HANDLER'. ImportError: No module named 'meiduo_api.utils'.
ERROR 2018-08-21 15:00:43,900 basehttp 124 "GET /authorizations/ HTTP/1.1" 500 120925
ERROR 2018-08-21 15:00:45,978 exception 135 Internal Server Error: /authorizations/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 157, in http_method_not_allowed
    raise exceptions.MethodNotAllowed(request.method)
rest_framework.exceptions.MethodNotAllowed: 方法 “GET” 不被允许。

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 183, in import_from_string
    module = import_module(module_path)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 944, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 222, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 986, in _gcd_import
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 956, in _find_and_load_unlocked
ImportError: No module named 'meiduo_api.utils'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 437, in handle_exception
    exception_handler = self.get_exception_handler()
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 288, in get_exception_handler
    return self.settings.EXCEPTION_HANDLER
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 227, in __getattr__
    val = perform_import(val, attr)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 170, in perform_import
    return import_from_string(val, setting_name)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/settings.py", line 187, in import_from_string
    raise ImportError(msg)
ImportError: Could not import 'meiduo_api.utils.exceptions.exception_handler' for API setting 'EXCEPTION_HANDLER'. ImportError: No module named 'meiduo_api.utils'.
ERROR 2018-08-21 15:00:46,068 basehttp 124 "GET /authorizations/ HTTP/1.1" 500 120925
WARNING 2018-08-21 15:01:58,928 basehttp 124 "GET /authorizations/ HTTP/1.1" 405 7589
INFO 2018-08-21 15:01:58,950 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 200 121200
INFO 2018-08-21 15:01:58,952 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 200 3385
INFO 2018-08-21 15:01:58,953 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 200 817
INFO 2018-08-21 15:01:58,956 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 200 1131
INFO 2018-08-21 15:01:58,974 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 200 86927
INFO 2018-08-21 15:01:58,975 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 200 37045
INFO 2018-08-21 15:01:58,981 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 200 1735
INFO 2018-08-21 15:01:58,985 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 200 3597
INFO 2018-08-21 15:01:59,002 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 200 13632
INFO 2018-08-21 15:01:59,006 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 200 1262
INFO 2018-08-21 15:01:59,008 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 200 1458
INFO 2018-08-21 15:02:43,059 basehttp 124 "POST /authorizations/ HTTP/1.1" 200 7709
INFO 2018-08-21 15:06:37,065 basehttp 124 "POST /authorizations/ HTTP/1.1" 200 7709
INFO 2018-08-21 15:08:50,300 basehttp 124 "POST /authorizations/ HTTP/1.1" 200 7787
INFO 2018-08-21 15:09:07,888 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
INFO 2018-08-21 15:09:07,930 basehttp 124 "POST /authorizations/ HTTP/1.1" 200 216
INFO 2018-08-21 15:28:35,297 basehttp 124 "POST /authorizations/ HTTP/1.1" 200 7787
INFO 2018-08-21 15:33:30,491 basehttp 124 "POST /authorizations/ HTTP/1.1" 200 7787
WARNING 2018-08-21 15:33:47,014 basehttp 124 "POST /authorizations/ HTTP/1.1" 400 7638
WARNING 2018-08-21 15:34:20,266 basehttp 124 "POST /authorizations/ HTTP/1.1" 400 7639
WARNING 2018-08-21 19:50:44,846 basehttp 124 "GET /usernames/mrping1111/count/ HTTP/1.1" 400 25
WARNING 2018-08-21 19:50:47,653 basehttp 124 "GET /usernames/mrping1111/count/ HTTP/1.1" 400 25
WARNING 2018-08-21 19:50:52,297 basehttp 124 "GET /usernames/mrping1111/count/ HTTP/1.1" 400 25
WARNING 2018-08-21 19:50:54,954 basehttp 124 "GET /usernames/mrping1111/count/ HTTP/1.1" 400 25
INFO 2018-08-21 19:51:32,493 basehttp 124 "GET /usernames/mrping11111/count/ HTTP/1.1" 200 36
WARNING 2018-08-21 19:51:47,031 basehttp 124 "GET /usernames/mrping1111/count/ HTTP/1.1" 400 25
WARNING 2018-08-21 20:34:27,334 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:34:27,336 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:34:28,854 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:34:28,855 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:34:29,638 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:34:29,639 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:34:32,599 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:34:32,600 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:03,152 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:03,152 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:03,441 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:03,442 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:03,601 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:03,601 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:03,749 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:03,751 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:03,885 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:03,886 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:04,018 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:04,019 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:04,404 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:04,406 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:12,558 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:12,559 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:15,348 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:15,348 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:15,504 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:15,505 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:15,662 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:15,664 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:15,807 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:15,807 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 20:35:15,975 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 20:35:15,976 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 21:32:30,318 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 21:32:30,320 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-21 21:32:31,749 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-21 21:32:31,751 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-22 15:36:30,015 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-22 15:36:30,017 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-22 15:36:31,253 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-22 15:36:31,254 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-22 15:36:32,550 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-22 15:36:32,550 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-22 15:36:32,713 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-22 15:36:32,713 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-22 15:36:32,869 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-22 15:36:32,871 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-22 15:36:33,014 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-22 15:36:33,014 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
WARNING 2018-08-22 15:36:33,138 base 152 Not Found: /oauth/qq/authorization/
WARNING 2018-08-22 15:36:33,140 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 404 2914
INFO 2018-08-22 15:38:43,176 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:45,124 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:45,579 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:45,733 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:45,891 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:46,052 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:46,211 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:46,355 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:46,485 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:46,633 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:46,766 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:46,913 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:47,141 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:38:50,141 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:39:03,582 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:39:10,345 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:39:59,875 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 15:40:23,269 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
WARNING 2018-08-22 16:02:00,857 base 152 Not Found: /oauth/qq/user/
WARNING 2018-08-22 16:02:00,859 basehttp 124 "GET /oauth/qq/user/?code=3798FBC4958748AE2C056ED2D04944B9 HTTP/1.1" 404 3091
INFO 2018-08-22 16:28:14,641 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
WARNING 2018-08-22 16:28:30,736 base 152 Not Found: /oauth/qq/user/
WARNING 2018-08-22 16:28:30,739 basehttp 124 "GET /oauth/qq/user/?code=39C9F5F5329074D0BC508695EE3D4554 HTTP/1.1" 404 3091
WARNING 2018-08-22 16:28:46,588 base 152 Not Found: /oauth/qq/user/
WARNING 2018-08-22 16:28:46,589 basehttp 124 "GET /oauth/qq/user/?code=null HTTP/1.1" 404 3063
WARNING 2018-08-22 16:29:17,004 base 152 Not Found: /oauth/qq/user/
WARNING 2018-08-22 16:29:17,014 basehttp 124 "GET /oauth/qq/user/?code=null HTTP/1.1" 404 3063
WARNING 2018-08-22 16:31:11,159 base 152 Not Found: /oauth/qq/user/
WARNING 2018-08-22 16:31:11,160 basehttp 124 "GET /oauth/qq/user/?code=null HTTP/1.1" 404 3063
WARNING 2018-08-22 16:31:14,151 base 152 Not Found: /oauth/qq/user/
WARNING 2018-08-22 16:31:14,152 basehttp 124 "GET /oauth/qq/user/?code=null HTTP/1.1" 404 3063
WARNING 2018-08-22 16:31:16,503 base 152 Not Found: /oauth/qq/user/
WARNING 2018-08-22 16:31:16,505 basehttp 124 "GET /oauth/qq/user/?code=null HTTP/1.1" 404 3063
WARNING 2018-08-22 16:31:20,408 base 152 Not Found: /oauth/qq/user/
WARNING 2018-08-22 16:31:20,409 basehttp 124 "GET /oauth/qq/user/?code=null HTTP/1.1" 404 3063
INFO 2018-08-22 16:36:00,786 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
WARNING 2018-08-22 16:36:32,344 base 152 Not Found: /oauth/qq/user/
WARNING 2018-08-22 16:36:32,358 basehttp 124 "GET /oauth/qq/user/?code=51661928D47A2816706D89ED857FB3CD HTTP/1.1" 404 3091
WARNING 2018-08-22 17:28:26,720 base 152 Not Found: /oauth/qq/user/
WARNING 2018-08-22 17:28:26,728 basehttp 124 "GET /oauth/qq/user/?code=51661928D47A2816706D89ED857FB3CD HTTP/1.1" 404 3091
WARNING 2018-08-22 19:23:14,782 base 152 Not Found: /oauth/qq/user/
WARNING 2018-08-22 19:23:14,786 basehttp 124 "GET /oauth/qq/user/?code=null HTTP/1.1" 404 3063
INFO 2018-08-22 19:23:27,231 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 19:23:29,311 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 19:23:40,600 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
WARNING 2018-08-22 19:23:59,917 base 152 Not Found: /oauth/qq/user/
WARNING 2018-08-22 19:23:59,918 basehttp 124 "GET /oauth/qq/user/?code=9C28D8E5B7732BA87FA05F28D0AF11D3 HTTP/1.1" 404 3091
INFO 2018-08-22 19:24:33,611 basehttp 124 "GET /oauth/qq/user/?code=9C28D8E5B7732BA87FA05F28D0AF11D3 HTTP/1.1" 200 150
INFO 2018-08-22 19:25:26,506 basehttp 124 "GET /sms_code/15222641000/ HTTP/1.1" 200 16
INFO 2018-08-22 19:26:53,291 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 19:26:53,312 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 405 46
WARNING 2018-08-22 19:26:55,306 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 405 46
WARNING 2018-08-22 19:26:56,394 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 405 46
INFO 2018-08-22 19:57:12,531 basehttp 124 "GET /sms_code/15222641000/ HTTP/1.1" 200 16
INFO 2018-08-22 19:57:24,989 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 19:57:25,008 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 19:57:31,358 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 19:57:31,376 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
ERROR 2018-08-22 19:57:58,909 basehttp 124 "GET /oauth/qq/user/?code=9C28D8E5B7732BA87FA05F28D0AF11D3 HTTP/1.1" 503 28
INFO 2018-08-22 19:58:11,464 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 19:58:31,549 basehttp 124 "GET /oauth/qq/user/?code=E794297DD49EB84E1A26C3B3CE09A3EE HTTP/1.1" 200 150
INFO 2018-08-22 19:58:42,249 basehttp 124 "GET /sms_code/15222641000/ HTTP/1.1" 200 16
WARNING 2018-08-22 19:58:49,216 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 19:58:54,802 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 19:58:54,811 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 19:59:38,475 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 19:59:38,482 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 20:00:13,073 basehttp 124 "GET /sms_code/15222641000/ HTTP/1.1" 200 16
INFO 2018-08-22 20:00:35,861 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 20:00:35,889 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 20:03:57,602 basehttp 124 "GET /sms_code/15222641000/ HTTP/1.1" 200 16
INFO 2018-08-22 20:04:06,396 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 20:04:06,429 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 20:04:27,233 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 20:04:41,518 basehttp 124 "GET /oauth/qq/user/?code=850F52FE540207582260B030AA792A51 HTTP/1.1" 200 150
INFO 2018-08-22 20:04:51,034 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
WARNING 2018-08-22 20:04:57,935 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 20:05:03,040 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 20:05:03,050 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 20:05:53,836 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
INFO 2018-08-22 20:06:04,444 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 20:06:04,482 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 20:07:48,160 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 20:07:59,222 basehttp 124 "GET /oauth/qq/user/?code=01702BEBDD1F3B550B8EE7F42F7724FF HTTP/1.1" 200 150
INFO 2018-08-22 20:08:04,517 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
INFO 2018-08-22 20:08:14,620 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 20:08:14,625 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 20:09:11,132 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
INFO 2018-08-22 20:09:20,411 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
ERROR 2018-08-22 20:09:20,432 exception 135 Internal Server Error: /oauth/qq/user/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/workspace/Web/Django/meiduo/meiduo_api/apps/oauth/views.py", line 60, in post
    serializer.is_valid(raise_exception=True)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 236, in is_valid
    self._validated_data = self.run_validation(self.initial_data)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 437, in run_validation
    value = self.validate(value)
  File "/home/python/workspace/Web/Django/meiduo/meiduo_api/apps/oauth/serializers.py", line 36, in validate
    print("解密" + openid)
TypeError: Can't convert 'NoneType' object to str implicitly
ERROR 2018-08-22 20:09:20,501 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 500 18706
INFO 2018-08-22 20:10:18,049 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
ERROR 2018-08-22 20:10:22,776 exception 135 Internal Server Error: /oauth/qq/user/
Traceback (most recent call last):
  File "/usr/lib/python3.5/urllib/request.py", line 1254, in do_open
    h.request(req.get_method(), req.selector, req.data, headers)
  File "/usr/lib/python3.5/http/client.py", line 1106, in request
    self._send_request(method, url, body, headers)
  File "/usr/lib/python3.5/http/client.py", line 1151, in _send_request
    self.endheaders(body)
  File "/usr/lib/python3.5/http/client.py", line 1102, in endheaders
    self._send_output(message_body)
  File "/usr/lib/python3.5/http/client.py", line 934, in _send_output
    self.send(msg)
  File "/usr/lib/python3.5/http/client.py", line 877, in send
    self.connect()
  File "/usr/lib/python3.5/http/client.py", line 1252, in connect
    super().connect()
  File "/usr/lib/python3.5/http/client.py", line 849, in connect
    (self.host,self.port), self.timeout, self.source_address)
  File "/usr/lib/python3.5/socket.py", line 693, in create_connection
    for res in getaddrinfo(host, port, 0, SOCK_STREAM):
  File "/usr/lib/python3.5/socket.py", line 732, in getaddrinfo
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
socket.gaierror: [Errno -3] Temporary failure in name resolution

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/workspace/Web/Django/meiduo/meiduo_api/apps/oauth/views.py", line 31, in get
    access_token = oauth.get_access_token(code)
  File "/home/python/workspace/Web/Django/meiduo/meiduo_api/apps/oauth/utils.py", line 45, in get_access_token
    response = urlopen(url)
  File "/usr/lib/python3.5/urllib/request.py", line 163, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/lib/python3.5/urllib/request.py", line 466, in open
    response = self._open(req, data)
  File "/usr/lib/python3.5/urllib/request.py", line 484, in _open
    '_open', req)
  File "/usr/lib/python3.5/urllib/request.py", line 444, in _call_chain
    result = func(*args)
  File "/usr/lib/python3.5/urllib/request.py", line 1297, in https_open
    context=self._context, check_hostname=self._check_hostname)
  File "/usr/lib/python3.5/urllib/request.py", line 1256, in do_open
    raise URLError(err)
urllib.error.URLError: <urlopen error [Errno -3] Temporary failure in name resolution>
ERROR 2018-08-22 20:10:22,840 basehttp 124 "GET /oauth/qq/user/?code=01702BEBDD1F3B550B8EE7F42F7724FF HTTP/1.1" 500 20365
INFO 2018-08-22 20:10:28,760 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 20:10:29,168 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 20:10:37,959 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 20:11:22,614 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 20:11:33,891 basehttp 124 "GET /oauth/qq/user/?code=49E1A712F700DB21168AD7FD04179B54 HTTP/1.1" 200 150
INFO 2018-08-22 20:11:41,581 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
INFO 2018-08-22 20:11:51,750 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 20:11:51,758 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 20:15:17,969 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 20:15:27,419 basehttp 124 "GET /oauth/qq/user/?code=78245B8DB804B129C60F7CCAF57877EE HTTP/1.1" 200 150
INFO 2018-08-22 20:18:19,331 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 20:18:27,465 basehttp 124 "GET /oauth/qq/user/?code=3EC079866D4249FC0E306F0B73EFFBB7 HTTP/1.1" 200 150
INFO 2018-08-22 20:18:40,891 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
INFO 2018-08-22 20:18:48,581 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 20:18:48,588 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 20:57:28,769 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 20:57:49,479 basehttp 124 "GET /oauth/qq/user/?code=6FD9C40FC18FBA2778E9AEE67D02AEC6 HTTP/1.1" 200 150
INFO 2018-08-22 20:58:27,326 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
INFO 2018-08-22 20:58:33,742 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 20:58:33,749 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 21:00:16,677 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
WARNING 2018-08-22 21:00:24,714 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
WARNING 2018-08-22 21:00:25,617 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 21:10:04,826 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 21:10:22,041 basehttp 124 "GET /oauth/qq/user/?code=D120781D296B89115754CBDF0E6F5A02 HTTP/1.1" 200 150
INFO 2018-08-22 21:11:27,664 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
INFO 2018-08-22 21:11:37,259 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 21:11:37,268 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 46
INFO 2018-08-22 21:14:28,088 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 21:14:36,502 basehttp 124 "GET /oauth/qq/user/?code=870099322E79BEC7387AF36F3155B055 HTTP/1.1" 200 150
INFO 2018-08-22 21:15:34,850 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 21:15:47,164 basehttp 124 "GET /oauth/qq/user/?code=7DBDA8E502F299B13FD590D6238B2727 HTTP/1.1" 200 150
INFO 2018-08-22 21:17:19,005 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 21:17:34,793 basehttp 124 "GET /oauth/qq/user/?code=A9695D5B1665D2C2109C1B82FBC48AAE HTTP/1.1" 200 190
INFO 2018-08-22 21:18:17,848 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
INFO 2018-08-22 21:19:01,628 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
ERROR 2018-08-22 21:19:01,701 exception 135 Internal Server Error: /oauth/qq/user/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 485, in dispatch
    self.response = self.finalize_response(request, response, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 400, in finalize_response
    % type(response)
AssertionError: Expected a `Response`, `HttpResponse` or `HttpStreamingResponse` to be returned from the view, but received a `<class 'dict'>`
ERROR 2018-08-22 21:19:01,768 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 500 88342
INFO 2018-08-22 21:19:08,470 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
ERROR 2018-08-22 21:19:08,620 exception 135 Internal Server Error: /oauth/qq/user/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 485, in dispatch
    self.response = self.finalize_response(request, response, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 400, in finalize_response
    % type(response)
AssertionError: Expected a `Response`, `HttpResponse` or `HttpStreamingResponse` to be returned from the view, but received a `<class 'dict'>`
ERROR 2018-08-22 21:19:08,698 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 500 88603
INFO 2018-08-22 21:20:18,570 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
INFO 2018-08-22 21:20:25,729 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
INFO 2018-08-22 21:20:25,855 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 200 218
INFO 2018-08-22 21:22:58,077 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 21:23:07,881 basehttp 124 "GET /oauth/qq/user/?code=0C850514D13C617A1F12199AA6481AB7 HTTP/1.1" 200 190
INFO 2018-08-22 21:23:18,308 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
WARNING 2018-08-22 21:23:54,958 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 37
WARNING 2018-08-22 21:24:03,864 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 37
INFO 2018-08-22 21:24:12,928 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 21:24:13,057 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 37
INFO 2018-08-22 21:24:33,826 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
WARNING 2018-08-22 21:24:33,909 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 400 37
INFO 2018-08-22 21:25:55,208 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 21:25:57,397 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 21:26:36,525 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 21:26:45,375 basehttp 124 "GET /oauth/qq/user/?code=73D55D7F7D56AE6810AEFC41046D7DD6 HTTP/1.1" 200 190
INFO 2018-08-22 21:26:51,295 basehttp 124 "GET /sms_code/15222641027/ HTTP/1.1" 200 16
INFO 2018-08-22 21:27:00,319 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
INFO 2018-08-22 21:27:00,418 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 200 218
INFO 2018-08-22 21:30:45,427 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 21:30:52,768 basehttp 124 "GET /oauth/qq/user/?code=A1C70355668C6E4D4A4E7827F1F9442D HTTP/1.1" 200 218
INFO 2018-08-22 21:53:31,802 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-22 21:53:54,500 basehttp 124 "GET /oauth/qq/user/?code=7D604F40AFB18CE5A3F58A2C78F2006C HTTP/1.1" 200 190
INFO 2018-08-24 08:50:28,235 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:50:31,358 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:50:32,854 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:50:33,382 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:50:33,557 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:50:43,181 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:50:45,334 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:50:46,063 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:50:46,590 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:50:47,079 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:50:47,246 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:50:47,408 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:50:54,444 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:51:05,917 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:51:41,953 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 08:51:53,383 basehttp 124 "GET /oauth/qq/user/?code=1DEA585FA240823E1E04A1AB059BCA21 HTTP/1.1" 200 190
INFO 2018-08-24 08:52:42,426 basehttp 124 "GET /sms_code/15222641000/ HTTP/1.1" 200 16
INFO 2018-08-24 08:52:49,829 basehttp 124 "OPTIONS /oauth/qq/user/ HTTP/1.1" 200 0
INFO 2018-08-24 08:52:49,908 basehttp 124 "POST /oauth/qq/user/ HTTP/1.1" 200 218
INFO 2018-08-24 08:52:54,480 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
WARNING 2018-08-24 08:52:54,537 base 152 Not Found: /user/
WARNING 2018-08-24 08:52:54,540 basehttp 124 "GET /user/ HTTP/1.1" 404 2960
INFO 2018-08-24 14:39:10,090 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
WARNING 2018-08-24 14:39:10,118 base 152 Not Found: /user/
WARNING 2018-08-24 14:39:10,120 basehttp 124 "GET /user/ HTTP/1.1" 404 2960
WARNING 2018-08-24 14:39:31,728 base 152 Not Found: /emails/verification/
WARNING 2018-08-24 14:39:31,730 basehttp 124 "GET /emails/verification/ HTTP/1.1" 404 3005
WARNING 2018-08-24 14:39:35,108 base 152 Not Found: /emails/verification/
WARNING 2018-08-24 14:39:35,109 basehttp 124 "GET /emails/verification/ HTTP/1.1" 404 3005
INFO 2018-08-24 14:41:17,231 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
WARNING 2018-08-24 14:41:17,255 base 152 Not Found: /user/
WARNING 2018-08-24 14:41:17,265 basehttp 124 "GET /user/ HTTP/1.1" 404 2960
WARNING 2018-08-24 14:41:30,162 base 152 Not Found: /user/
WARNING 2018-08-24 14:41:30,162 basehttp 124 "GET /user/ HTTP/1.1" 404 2960
INFO 2018-08-24 14:57:43,662 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
INFO 2018-08-24 14:57:43,685 basehttp 124 "GET /user/ HTTP/1.1" 200 88
INFO 2018-08-24 14:57:43,692 basehttp 124 "OPTIONS /browse_histories/ HTTP/1.1" 200 0
WARNING 2018-08-24 14:57:43,701 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 14:57:43,703 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3151
INFO 2018-08-24 14:58:15,155 basehttp 124 "GET /oauth/qq/authorization/?next=/user_center_info.html HTTP/1.1" 200 196
INFO 2018-08-24 14:58:16,797 basehttp 124 "GET /oauth/qq/authorization/?next=/user_center_info.html HTTP/1.1" 200 196
INFO 2018-08-24 14:58:18,462 basehttp 124 "GET /oauth/qq/authorization/?next=/user_center_info.html HTTP/1.1" 200 196
INFO 2018-08-24 14:58:26,907 basehttp 124 "GET /oauth/qq/authorization/?next=/user_center_info.html HTTP/1.1" 200 196
INFO 2018-08-24 14:58:46,977 basehttp 124 "GET /oauth/qq/user/?code=47C608EC30C265B6BB7FF5156FB9743B HTTP/1.1" 200 218
INFO 2018-08-24 14:58:47,101 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
INFO 2018-08-24 14:58:47,143 basehttp 124 "GET /user/ HTTP/1.1" 200 88
INFO 2018-08-24 14:58:47,154 basehttp 124 "OPTIONS /browse_histories/ HTTP/1.1" 200 0
WARNING 2018-08-24 14:58:47,168 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 14:58:47,168 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3151
INFO 2018-08-24 14:58:56,799 basehttp 124 "GET /user/ HTTP/1.1" 200 88
WARNING 2018-08-24 14:58:56,836 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 14:58:56,837 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3151
INFO 2018-08-24 14:59:36,205 basehttp 124 "OPTIONS /emails/ HTTP/1.1" 200 0
WARNING 2018-08-24 14:59:36,215 base 152 Not Found: /emails/
WARNING 2018-08-24 14:59:36,216 basehttp 124 "PUT /emails/ HTTP/1.1" 404 3121
WARNING 2018-08-24 14:59:39,887 base 152 Not Found: /emails/
WARNING 2018-08-24 14:59:39,889 basehttp 124 "PUT /emails/ HTTP/1.1" 404 3121
INFO 2018-08-24 14:59:43,459 basehttp 124 "OPTIONS /emails/ HTTP/1.1" 200 0
WARNING 2018-08-24 14:59:43,465 base 152 Not Found: /emails/
WARNING 2018-08-24 14:59:43,470 basehttp 124 "PUT /emails/ HTTP/1.1" 404 3121
INFO 2018-08-24 15:25:06,915 basehttp 124 "OPTIONS /emails/ HTTP/1.1" 200 0
ERROR 2018-08-24 15:25:07,007 exception 135 Internal Server Error: /emails/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/generics.py", line 228, in put
    return self.update(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/mixins.py", line 69, in update
    serializer.is_valid(raise_exception=True)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 236, in is_valid
    self._validated_data = self.run_validation(self.initial_data)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 434, in run_validation
    value = self.to_internal_value(data)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 482, in to_internal_value
    fields = self._writable_fields
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 370, in _writable_fields
    field for field in self.fields.values() if not field.read_only
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 363, in fields
    for key, value in self.get_fields().items():
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 1019, in get_fields
    info = model_meta.get_field_info(model)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/utils/model_meta.py", line 35, in get_field_info
    opts = model._meta.concrete_model._meta
AttributeError: type object 'UserDetailSerializer' has no attribute '_meta'
ERROR 2018-08-24 15:25:07,067 basehttp 124 "PUT /emails/ HTTP/1.1" 500 20056
INFO 2018-08-24 15:27:26,144 basehttp 124 "OPTIONS /emails/ HTTP/1.1" 200 0
ERROR 2018-08-24 15:27:26,199 exception 135 Internal Server Error: /emails/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/generics.py", line 228, in put
    return self.update(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/mixins.py", line 70, in update
    self.perform_update(serializer)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/mixins.py", line 80, in perform_update
    serializer.save()
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 209, in save
    self.instance = self.update(self.instance, validated_data)
  File "/home/python/workspace/Web/Django/meiduo/meiduo_api/apps/users/serializers.py", line 117, in update
    verify_url = instance.generate_verify_email_url()
  File "/home/python/workspace/Web/Django/meiduo/meiduo_api/apps/users/models.py", line 22, in generate_verify_email_url
    verify_url = 'http://www.meiduo.site:8080/success_verify_email.html?token=' + token
TypeError: Can't convert 'bytes' object to str implicitly
ERROR 2018-08-24 15:27:26,229 basehttp 124 "PUT /emails/ HTTP/1.1" 500 19260
WARNING 2018-08-24 15:27:49,036 base 152 Not Found: /emails/verification/
WARNING 2018-08-24 15:27:49,040 basehttp 124 "GET /emails/verification/?token= HTTP/1.1" 404 3324
WARNING 2018-08-24 15:27:49,080 base 152 Not Found: /emails/verification/
WARNING 2018-08-24 15:27:49,082 basehttp 124 "GET /emails/verification/?token= HTTP/1.1" 404 3324
INFO 2018-08-24 15:28:14,399 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
INFO 2018-08-24 15:28:14,426 basehttp 124 "GET /user/ HTTP/1.1" 200 104
INFO 2018-08-24 15:28:14,437 basehttp 124 "OPTIONS /browse_histories/ HTTP/1.1" 200 0
WARNING 2018-08-24 15:28:14,456 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 15:28:14,457 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3308
INFO 2018-08-24 15:29:40,585 basehttp 124 "OPTIONS /emails/ HTTP/1.1" 200 0
INFO 2018-08-24 15:29:40,694 basehttp 124 "PUT /emails/ HTTP/1.1" 200 35
INFO 2018-08-24 15:37:12,373 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
INFO 2018-08-24 15:37:12,460 basehttp 124 "GET /user/ HTTP/1.1" 200 104
INFO 2018-08-24 15:37:12,500 basehttp 124 "OPTIONS /browse_histories/ HTTP/1.1" 200 0
WARNING 2018-08-24 15:37:12,515 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 15:37:12,527 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3308
INFO 2018-08-24 15:37:17,972 basehttp 124 "OPTIONS /emails/ HTTP/1.1" 200 0
INFO 2018-08-24 15:37:18,023 basehttp 124 "PUT /emails/ HTTP/1.1" 200 35
INFO 2018-08-24 15:38:39,747 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
INFO 2018-08-24 15:38:39,812 basehttp 124 "GET /user/ HTTP/1.1" 200 104
INFO 2018-08-24 15:38:39,844 basehttp 124 "OPTIONS /browse_histories/ HTTP/1.1" 200 0
WARNING 2018-08-24 15:38:39,872 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 15:38:39,899 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3308
INFO 2018-08-24 15:38:40,871 basehttp 124 "OPTIONS /emails/ HTTP/1.1" 200 0
INFO 2018-08-24 15:38:40,971 basehttp 124 "PUT /emails/ HTTP/1.1" 200 35
INFO 2018-08-24 15:47:42,614 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
INFO 2018-08-24 15:47:42,681 basehttp 124 "GET /user/ HTTP/1.1" 200 104
INFO 2018-08-24 15:47:42,700 basehttp 124 "OPTIONS /browse_histories/ HTTP/1.1" 200 0
WARNING 2018-08-24 15:47:42,722 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 15:47:42,724 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3308
INFO 2018-08-24 15:47:46,161 basehttp 124 "PUT /emails/ HTTP/1.1" 200 35
INFO 2018-08-24 15:58:00,955 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
INFO 2018-08-24 15:58:01,006 basehttp 124 "GET /user/ HTTP/1.1" 200 104
INFO 2018-08-24 15:58:01,030 basehttp 124 "OPTIONS /browse_histories/ HTTP/1.1" 200 0
WARNING 2018-08-24 15:58:01,040 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 15:58:01,043 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3308
INFO 2018-08-24 15:58:03,694 basehttp 124 "OPTIONS /emails/ HTTP/1.1" 200 0
INFO 2018-08-24 15:58:03,750 basehttp 124 "PUT /emails/ HTTP/1.1" 200 35
INFO 2018-08-24 15:59:50,068 basehttp 124 "GET /user/ HTTP/1.1" 200 104
WARNING 2018-08-24 15:59:50,110 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 15:59:50,110 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3308
INFO 2018-08-24 16:00:16,073 basehttp 124 "OPTIONS /addresses/ HTTP/1.1" 200 0
WARNING 2018-08-24 16:00:16,081 base 152 Not Found: /areas/
WARNING 2018-08-24 16:00:16,082 basehttp 124 "GET /areas/ HTTP/1.1" 404 3275
WARNING 2018-08-24 16:00:16,130 base 152 Not Found: /addresses/
WARNING 2018-08-24 16:00:16,131 basehttp 124 "GET /addresses/ HTTP/1.1" 404 3287
INFO 2018-08-24 16:00:21,420 basehttp 124 "GET /user/ HTTP/1.1" 200 104
WARNING 2018-08-24 16:00:21,441 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 16:00:21,447 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3308
INFO 2018-08-24 16:00:52,919 basehttp 124 "OPTIONS /authorizations/ HTTP/1.1" 200 0
WARNING 2018-08-24 16:00:52,960 basehttp 124 "POST /authorizations/ HTTP/1.1" 400 67
INFO 2018-08-24 16:00:58,660 basehttp 124 "POST /authorizations/ HTTP/1.1" 200 218
INFO 2018-08-24 16:01:02,063 basehttp 124 "GET /user/ HTTP/1.1" 200 88
WARNING 2018-08-24 16:01:02,105 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 16:01:02,106 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3308
INFO 2018-08-24 16:01:12,867 basehttp 124 "PUT /emails/ HTTP/1.1" 200 36
INFO 2018-08-24 16:03:20,997 basehttp 124 "GET /user/ HTTP/1.1" 200 105
WARNING 2018-08-24 16:03:21,019 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 16:03:21,022 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3308
WARNING 2018-08-24 16:04:41,563 base 152 Not Found: /emails/verification/
WARNING 2018-08-24 16:04:41,601 basehttp 124 "GET /emails/verification/?token=eyJleHAiOjE1MzUxODM4ODMsImFsZyI6IkhTMjU2IiwiaWF0IjoxNTM1MDk3NDgzfQ.eyJlbWFpbCI6InlwaW5naG9vQDE2My5jb20iLCJ1c2VyX2lkIjo2fQ.NGogbrQO7h0xOohq-jpE-Yixhsj-zUXeB5XeYQXkiGA HTTP/1.1" 404 3489
INFO 2018-08-24 16:17:43,845 basehttp 124 "GET /emails/verification/?token=eyJleHAiOjE1MzUxODM4ODMsImFsZyI6IkhTMjU2IiwiaWF0IjoxNTM1MDk3NDgzfQ.eyJlbWFpbCI6InlwaW5naG9vQDE2My5jb20iLCJ1c2VyX2lkIjo2fQ.NGogbrQO7h0xOohq-jpE-Yixhsj-zUXeB5XeYQXkiGA HTTP/1.1" 200 16
INFO 2018-08-24 16:17:48,052 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
INFO 2018-08-24 16:17:48,149 basehttp 124 "GET /user/ HTTP/1.1" 200 105
INFO 2018-08-24 16:17:48,163 basehttp 124 "OPTIONS /browse_histories/ HTTP/1.1" 200 0
WARNING 2018-08-24 16:17:48,183 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 16:17:48,184 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3478
INFO 2018-08-24 16:17:50,646 basehttp 124 "GET /user/ HTTP/1.1" 200 105
WARNING 2018-08-24 16:17:50,663 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 16:17:50,664 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3478
INFO 2018-08-24 16:17:54,243 basehttp 124 "OPTIONS /emails/ HTTP/1.1" 200 0
INFO 2018-08-24 16:17:54,344 basehttp 124 "PUT /emails/ HTTP/1.1" 200 36
INFO 2018-08-24 16:18:34,918 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-24 16:18:46,362 basehttp 124 "GET /oauth/qq/user/?code=86722CCF3AD167D3CB6216B06CEB99F8 HTTP/1.1" 200 240
INFO 2018-08-24 16:18:48,202 basehttp 124 "GET /user/ HTTP/1.1" 200 103
WARNING 2018-08-24 16:18:48,248 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 16:18:48,250 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3478
INFO 2018-08-24 16:18:55,104 basehttp 124 "GET /user/ HTTP/1.1" 200 103
WARNING 2018-08-24 16:18:55,126 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 16:18:55,128 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3478
INFO 2018-08-24 16:40:22,727 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
INFO 2018-08-24 16:40:22,815 basehttp 124 "GET /user/ HTTP/1.1" 200 103
INFO 2018-08-24 16:40:22,833 basehttp 124 "OPTIONS /browse_histories/ HTTP/1.1" 200 0
WARNING 2018-08-24 16:40:22,860 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 16:40:22,861 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 3478
WARNING 2018-08-24 17:10:29,387 base 152 Not Found: /areas/
INFO 2018-08-24 17:10:29,387 basehttp 124 "OPTIONS /addresses/ HTTP/1.1" 200 0
WARNING 2018-08-24 17:10:29,393 base 152 Not Found: /addresses/
WARNING 2018-08-24 17:10:29,416 basehttp 124 "GET /addresses/ HTTP/1.1" 404 3457
WARNING 2018-08-24 17:10:29,419 basehttp 124 "GET /areas/ HTTP/1.1" 404 3445
INFO 2018-08-24 18:54:41,095 basehttp 124 "OPTIONS /addresses/ HTTP/1.1" 200 0
WARNING 2018-08-24 18:54:41,111 base 152 Not Found: /addresses/
WARNING 2018-08-24 18:54:41,127 basehttp 124 "GET /addresses/ HTTP/1.1" 404 4628
ERROR 2018-08-24 18:54:41,179 exception 135 Internal Server Error: /areas/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/mixins.py", line 48, in list
    return Response(serializer.data)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 765, in data
    ret = super(ListSerializer, self).data
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 262, in data
    self._data = self.to_representation(self.instance)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 683, in to_representation
    self.child.to_representation(item) for item in iterable
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 683, in <listcomp>
    self.child.to_representation(item) for item in iterable
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 510, in to_representation
    fields = self._readable_fields
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 376, in _readable_fields
    field for field in self.fields.values()
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 363, in fields
    for key, value in self.get_fields().items():
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 1020, in get_fields
    field_names = self.get_field_names(declared_fields, info)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 1097, in get_field_names
    serializer_class=self.__class__.__name__
AssertionError: ("Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the AreaSerializer serializer.",)
ERROR 2018-08-24 18:54:41,232 basehttp 124 "GET /areas/ HTTP/1.1" 500 20394
INFO 2018-08-24 18:54:54,517 basehttp 124 "GET / HTTP/1.1" 200 5183
INFO 2018-08-24 18:54:54,539 basehttp 124 "GET /static/rest_framework/css/default.css HTTP/1.1" 304 0
INFO 2018-08-24 18:54:54,540 basehttp 124 "GET /static/rest_framework/css/bootstrap-tweaks.css HTTP/1.1" 304 0
INFO 2018-08-24 18:54:54,541 basehttp 124 "GET /static/rest_framework/css/bootstrap.min.css HTTP/1.1" 304 0
INFO 2018-08-24 18:54:54,541 basehttp 124 "GET /static/rest_framework/js/ajax-form.js HTTP/1.1" 304 0
INFO 2018-08-24 18:54:54,541 basehttp 124 "GET /static/rest_framework/js/jquery-3.3.1.min.js HTTP/1.1" 304 0
INFO 2018-08-24 18:54:54,542 basehttp 124 "GET /static/rest_framework/css/prettify.css HTTP/1.1" 304 0
INFO 2018-08-24 18:54:54,548 basehttp 124 "GET /static/rest_framework/js/csrf.js HTTP/1.1" 304 0
INFO 2018-08-24 18:54:54,551 basehttp 124 "GET /static/rest_framework/js/bootstrap.min.js HTTP/1.1" 304 0
INFO 2018-08-24 18:54:54,566 basehttp 124 "GET /static/rest_framework/js/prettify-min.js HTTP/1.1" 304 0
INFO 2018-08-24 18:54:54,567 basehttp 124 "GET /static/rest_framework/js/default.js HTTP/1.1" 304 0
INFO 2018-08-24 18:54:54,635 basehttp 124 "GET /static/rest_framework/img/grid.png HTTP/1.1" 304 0
WARNING 2018-08-24 18:54:54,824 base 152 Not Found: /favicon.ico
WARNING 2018-08-24 18:54:54,825 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 4631
ERROR 2018-08-24 18:54:56,686 exception 135 Internal Server Error: /areas/
Traceback (most recent call last):
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/viewsets.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 483, in dispatch
    response = self.handle_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 443, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/views.py", line 480, in dispatch
    response = handler(request, *args, **kwargs)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/mixins.py", line 48, in list
    return Response(serializer.data)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 765, in data
    ret = super(ListSerializer, self).data
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 262, in data
    self._data = self.to_representation(self.instance)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 683, in to_representation
    self.child.to_representation(item) for item in iterable
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 683, in <listcomp>
    self.child.to_representation(item) for item in iterable
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 510, in to_representation
    fields = self._readable_fields
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/django/utils/functional.py", line 35, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 376, in _readable_fields
    field for field in self.fields.values()
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 363, in fields
    for key, value in self.get_fields().items():
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 1020, in get_fields
    field_names = self.get_field_names(declared_fields, info)
  File "/home/python/.virtualenvs/py3_django/lib/python3.5/site-packages/rest_framework/serializers.py", line 1097, in get_field_names
    serializer_class=self.__class__.__name__
AssertionError: ("Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the AreaSerializer serializer.",)
ERROR 2018-08-24 18:54:56,745 basehttp 124 "GET /areas/ HTTP/1.1" 500 151302
INFO 2018-08-24 18:55:55,239 basehttp 124 "GET /areas/ HTTP/1.1" 200 8424
INFO 2018-08-24 18:56:01,783 basehttp 124 "GET /areas/110000 HTTP/1.1" 301 0
INFO 2018-08-24 18:56:01,824 basehttp 124 "GET /areas/110000/ HTTP/1.1" 200 5569
INFO 2018-08-24 18:56:17,856 basehttp 124 "GET /areas HTTP/1.1" 301 0
INFO 2018-08-24 18:56:17,906 basehttp 124 "GET /areas/ HTTP/1.1" 200 8424
INFO 2018-08-24 18:56:24,409 basehttp 124 "GET /areas/140000 HTTP/1.1" 301 0
INFO 2018-08-24 18:56:24,483 basehttp 124 "GET /areas/140000/ HTTP/1.1" 200 6659
WARNING 2018-08-24 18:58:45,340 base 152 Not Found: /areas/140000/140100
WARNING 2018-08-24 18:58:45,346 basehttp 124 "GET /areas/140000/140100 HTTP/1.1" 404 4655
WARNING 2018-08-24 18:59:11,071 base 152 Not Found: /areas/140000/reas-detail
WARNING 2018-08-24 18:59:11,071 basehttp 124 "GET /areas/140000/reas-detail HTTP/1.1" 404 4670
WARNING 2018-08-24 18:59:20,916 base 152 Not Found: /areas/140000/areas-detail
WARNING 2018-08-24 18:59:20,917 basehttp 124 "GET /areas/140000/areas-detail HTTP/1.1" 404 4673
INFO 2018-08-24 18:59:25,257 basehttp 124 "GET /areas/140000/ HTTP/1.1" 200 6659
INFO 2018-08-24 19:00:51,284 basehttp 124 "GET /areas/140000/ HTTP/1.1" 200 6659
INFO 2018-08-24 19:00:56,008 basehttp 124 "GET /areas/140000/?format=json HTTP/1.1" 200 404
WARNING 2018-08-24 19:01:28,855 base 152 Not Found: /addresses/
WARNING 2018-08-24 19:01:28,855 basehttp 124 "GET /addresses/ HTTP/1.1" 404 4628
INFO 2018-08-24 19:01:28,859 basehttp 124 "GET /areas/ HTTP/1.1" 200 1204
WARNING 2018-08-24 19:01:35,743 base 152 Not Found: /addresses/
WARNING 2018-08-24 19:01:35,747 basehttp 124 "GET /addresses/ HTTP/1.1" 404 4628
INFO 2018-08-24 19:01:35,751 basehttp 124 "GET /areas/ HTTP/1.1" 200 1204
INFO 2018-08-24 19:12:59,386 basehttp 124 "GET / HTTP/1.1" 200 5183
INFO 2018-08-24 19:13:04,232 basehttp 124 "GET /?/admin HTTP/1.1" 200 5232
INFO 2018-08-24 19:13:14,048 basehttp 124 "GET /admin HTTP/1.1" 301 0
INFO 2018-08-24 19:13:14,066 basehttp 124 "GET /admin/ HTTP/1.1" 302 0
INFO 2018-08-24 19:13:14,150 basehttp 124 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 1652
INFO 2018-08-24 19:13:14,173 basehttp 124 "GET /static/admin/css/login.css HTTP/1.1" 200 1203
INFO 2018-08-24 19:13:14,174 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 200 16066
INFO 2018-08-24 19:13:14,300 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 200 423
INFO 2018-08-24 19:13:14,321 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 80304
INFO 2018-08-24 19:13:14,325 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 81348
INFO 2018-08-24 19:13:36,969 basehttp 124 "POST /admin/login/?next=/admin/ HTTP/1.1" 200 1783
INFO 2018-08-24 19:13:36,999 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 200 82564
INFO 2018-08-24 19:13:46,932 basehttp 124 "POST /admin/login/?next=/admin/ HTTP/1.1" 200 1789
INFO 2018-08-24 19:13:51,128 basehttp 124 "POST /admin/login/?next=/admin/ HTTP/1.1" 200 1789
INFO 2018-08-24 19:14:02,416 basehttp 124 "POST /admin/login/?next=/admin/ HTTP/1.1" 200 1791
INFO 2018-08-24 19:17:23,462 basehttp 124 "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2018-08-24 19:17:23,509 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:17:23,520 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 200 412
INFO 2018-08-24 19:17:23,537 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
INFO 2018-08-24 19:17:23,538 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
INFO 2018-08-24 19:17:27,139 basehttp 124 "GET /admin/auth/group/ HTTP/1.1" 200 3359
INFO 2018-08-24 19:17:27,162 basehttp 124 "GET /static/admin/css/changelists.css HTTP/1.1" 200 6170
INFO 2018-08-24 19:17:27,165 basehttp 124 "GET /static/admin/js/core.js HTTP/1.1" 200 7976
INFO 2018-08-24 19:17:27,185 basehttp 124 "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 363
INFO 2018-08-24 19:17:27,186 basehttp 124 "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 6591
INFO 2018-08-24 19:17:27,188 basehttp 124 "GET /static/admin/js/actions.js HTTP/1.1" 200 6501
INFO 2018-08-24 19:17:27,189 basehttp 124 "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 258648
INFO 2018-08-24 19:17:27,210 basehttp 124 "GET /static/admin/js/urlify.js HTTP/1.1" 200 8186
INFO 2018-08-24 19:17:27,214 basehttp 124 "GET /static/admin/img/search.svg HTTP/1.1" 200 458
INFO 2018-08-24 19:17:27,242 basehttp 124 "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1538
INFO 2018-08-24 19:17:27,256 basehttp 124 "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 128820
INFO 2018-08-24 19:17:27,259 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7682
INFO 2018-08-24 19:17:27,389 basehttp 124 "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
INFO 2018-08-24 19:17:29,013 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:19:33,408 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:19:33,525 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:19:33,528 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:19:33,558 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:19:33,572 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:19:33,587 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:19:33,589 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:19:33,591 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:19:33,592 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:19:44,589 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:19:44,685 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:19:44,686 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:19:44,690 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:19:44,699 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:19:44,703 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:19:44,725 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:19:44,733 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:19:44,734 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:19:51,120 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:19:52,549 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:34,018 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:34,039 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:34,051 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:34,063 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:34,084 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:34,087 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:34,094 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:34,095 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:34,097 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:35,385 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:35,406 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:35,407 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:35,423 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:35,429 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:35,440 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:35,446 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:35,449 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:35,452 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,010 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:36,040 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,041 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,052 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,060 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,063 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,086 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,087 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,099 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,199 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:36,212 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,218 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,220 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,228 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,231 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,244 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,246 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,266 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,429 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:36,481 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,482 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,488 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,503 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,504 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,524 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,530 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,531 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,718 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:36,737 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,739 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,747 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,761 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,768 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,773 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,779 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,783 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,956 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:36,976 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,978 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:36,991 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,012 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,019 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,045 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,051 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,054 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,156 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:37,187 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,189 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,195 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,207 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,209 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,221 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,229 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,258 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,455 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:37,496 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,501 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,503 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,516 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,518 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,543 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,544 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,545 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,626 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:37,640 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,641 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,649 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,658 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,670 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,692 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,697 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,712 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,787 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:37,803 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,804 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,806 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,812 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,818 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,833 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,840 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,848 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,927 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:37,949 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,950 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,960 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,965 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,982 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:37,993 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,010 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,014 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,079 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:38,094 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,095 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,102 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,107 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,109 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,113 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,123 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,132 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,233 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:38,249 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,249 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,258 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,268 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,274 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,293 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,299 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,307 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,420 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:38,435 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,435 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,443 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,449 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,452 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,457 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,459 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,465 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,741 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:38,754 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,755 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,760 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,765 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,770 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,776 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,780 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,787 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,943 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:38,962 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,964 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,973 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,980 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,982 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,989 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:38,994 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:39,012 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:39,116 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:20:39,125 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:39,125 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:39,130 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:20:39,140 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:39,143 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:39,146 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:20:39,148 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:39,149 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:20:41,615 basehttp 124 "GET /admin/ HTTP/1.1" 200 2354
INFO 2018-08-24 19:21:10,793 basehttp 124 "GET /admin/ HTTP/1.1" 200 2986
INFO 2018-08-24 19:21:10,856 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:21:10,857 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:21:10,865 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:21:10,874 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:21:10,876 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:21:10,900 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:21:10,948 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:21:10,949 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:21:50,460 basehttp 124 "GET /admin/ HTTP/1.1" 200 3624
INFO 2018-08-24 19:21:50,556 basehttp 124 "GET /static/admin/css/base.css HTTP/1.1" 304 0
INFO 2018-08-24 19:21:50,556 basehttp 124 "GET /static/admin/css/dashboard.css HTTP/1.1" 304 0
INFO 2018-08-24 19:21:50,574 basehttp 124 "GET /static/admin/css/fonts.css HTTP/1.1" 304 0
INFO 2018-08-24 19:21:50,584 basehttp 124 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:21:50,585 basehttp 124 "GET /static/admin/fonts/Roboto-Bold-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:21:50,590 basehttp 124 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 304 0
INFO 2018-08-24 19:21:50,593 basehttp 124 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:21:50,594 basehttp 124 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 304 0
INFO 2018-08-24 19:22:04,039 basehttp 124 "GET /admin/areas/area/ HTTP/1.1" 200 27805
INFO 2018-08-24 19:22:04,086 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7682
INFO 2018-08-24 19:22:12,033 basehttp 124 "GET /admin/areas/area/820000/change/ HTTP/1.1" 200 157074
INFO 2018-08-24 19:22:12,076 basehttp 124 "GET /static/admin/css/forms.css HTTP/1.1" 200 8173
INFO 2018-08-24 19:22:12,095 basehttp 124 "GET /static/admin/css/widgets.css HTTP/1.1" 200 10340
INFO 2018-08-24 19:22:12,126 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7682
INFO 2018-08-24 19:22:12,277 basehttp 124 "GET /static/admin/js/change_form.js HTTP/1.1" 200 712
INFO 2018-08-24 19:22:12,284 basehttp 124 "GET /static/admin/img/icon-deletelink.svg HTTP/1.1" 200 392
INFO 2018-08-24 19:22:12,289 basehttp 124 "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 495
INFO 2018-08-24 19:22:15,196 basehttp 124 "GET /admin/areas/area/ HTTP/1.1" 200 27805
INFO 2018-08-24 19:48:35,580 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
INFO 2018-08-24 19:48:35,694 basehttp 124 "GET /user/ HTTP/1.1" 200 103
INFO 2018-08-24 19:48:35,707 basehttp 124 "OPTIONS /browse_histories/ HTTP/1.1" 200 0
WARNING 2018-08-24 19:48:35,723 base 152 Not Found: /browse_histories/
WARNING 2018-08-24 19:48:35,724 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 4649
INFO 2018-08-24 19:48:37,587 basehttp 124 "OPTIONS /addresses/ HTTP/1.1" 200 0
INFO 2018-08-24 19:48:37,587 basehttp 124 "GET /areas/ HTTP/1.1" 200 1204
WARNING 2018-08-24 19:48:37,623 base 152 Not Found: /addresses/
WARNING 2018-08-24 19:48:37,625 basehttp 124 "GET /addresses/ HTTP/1.1" 404 4628
INFO 2018-08-24 19:48:46,636 basehttp 124 "GET /areas/120000/ HTTP/1.1" 200 74
INFO 2018-08-24 19:48:48,893 basehttp 124 "GET /areas/120100/ HTTP/1.1" 200 572
INFO 2018-08-24 19:49:09,043 basehttp 124 "GET /areas/150000/ HTTP/1.1" 200 485
INFO 2018-08-24 19:49:10,775 basehttp 124 "GET /areas/150400/ HTTP/1.1" 200 467
INFO 2018-08-24 19:49:21,920 basehttp 124 "GET /areas/620000/ HTTP/1.1" 200 530
INFO 2018-08-24 19:49:24,666 basehttp 124 "GET /areas/621200/ HTTP/1.1" 200 323
INFO 2018-08-24 20:06:38,645 basehttp 124 "GET / HTTP/1.1" 200 5218
WARNING 2018-08-24 20:06:38,814 base 152 Not Found: /favicon.ico
WARNING 2018-08-24 20:06:38,814 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 4631
INFO 2018-08-24 20:06:41,564 basehttp 124 "GET /admin/ HTTP/1.1" 200 3624
INFO 2018-08-24 20:06:43,829 basehttp 124 "GET /admin/areas/area/ HTTP/1.1" 200 27956
INFO 2018-08-24 20:06:43,857 basehttp 124 "GET /admin/jsi18n/ HTTP/1.1" 200 7682
INFO 2018-08-24 20:32:34,827 basehttp 124 "GET /areas/710000/ HTTP/1.1" 200 42
INFO 2018-08-24 20:32:45,128 basehttp 124 "GET /areas/650000/ HTTP/1.1" 200 698
INFO 2018-08-24 20:32:53,154 basehttp 124 "GET /areas/710000/ HTTP/1.1" 200 42
INFO 2018-08-24 20:43:04,278 basehttp 124 "GET /areas/420000/ HTTP/1.1" 200 542
INFO 2018-08-24 20:43:05,382 basehttp 124 "GET /areas/420800/ HTTP/1.1" 200 206
INFO 2018-08-24 20:43:12,770 basehttp 124 "GET /areas/150000/ HTTP/1.1" 200 485
INFO 2018-08-24 20:43:15,242 basehttp 124 "GET /areas/150400/ HTTP/1.1" 200 467
INFO 2018-08-24 20:43:18,470 basehttp 124 "GET /areas/230000/ HTTP/1.1" 200 503
INFO 2018-08-24 20:43:20,782 basehttp 124 "GET /areas/230500/ HTTP/1.1" 200 311
INFO 2018-08-24 20:43:22,782 basehttp 124 "GET /areas/230900/ HTTP/1.1" 200 179
INFO 2018-08-24 20:43:25,767 basehttp 124 "GET /areas/360000/ HTTP/1.1" 200 407
INFO 2018-08-24 20:43:27,120 basehttp 124 "GET /areas/360800/ HTTP/1.1" 200 473
INFO 2018-08-24 20:43:32,831 basehttp 124 "GET /areas/130000/ HTTP/1.1" 200 464
INFO 2018-08-24 20:43:48,467 basehttp 124 "OPTIONS /addresses/ HTTP/1.1" 200 0
WARNING 2018-08-24 20:43:48,513 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:48,514 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:49,449 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:49,450 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:49,624 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:49,625 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:49,782 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:49,783 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:49,949 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:49,950 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:50,109 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:50,110 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:50,263 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:50,263 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:50,413 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:50,414 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:50,561 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:50,562 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:50,701 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:50,702 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:50,842 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:50,844 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:51,133 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:51,134 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:51,301 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:51,301 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:51,463 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:51,463 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
WARNING 2018-08-24 20:43:51,580 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:51,582 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
INFO 2018-08-24 20:43:58,667 basehttp 124 "OPTIONS /addresses/ HTTP/1.1" 200 0
WARNING 2018-08-24 20:43:58,691 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:43:58,695 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
INFO 2018-08-24 20:44:20,528 basehttp 124 "GET /areas/130500/ HTTP/1.1" 200 662
INFO 2018-08-24 20:44:21,538 basehttp 124 "OPTIONS /addresses/ HTTP/1.1" 200 0
WARNING 2018-08-24 20:44:21,555 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:44:21,563 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
INFO 2018-08-24 20:44:24,811 basehttp 124 "OPTIONS /addresses/ HTTP/1.1" 200 0
WARNING 2018-08-24 20:44:24,827 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:44:24,828 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
INFO 2018-08-24 20:44:25,101 basehttp 124 "OPTIONS /addresses/ HTTP/1.1" 200 0
WARNING 2018-08-24 20:44:25,120 base 152 Not Found: /addresses/
WARNING 2018-08-24 20:44:25,120 basehttp 124 "POST /addresses/ HTTP/1.1" 404 4629
INFO 2018-08-24 20:59:15,800 basehttp 124 "GET /areas/710000/ HTTP/1.1" 200 42
INFO 2018-08-24 21:18:17,768 basehttp 124 "OPTIONS /addresses/ HTTP/1.1" 200 0
WARNING 2018-08-24 21:18:17,791 base 152 Not Found: /addresses/
WARNING 2018-08-24 21:18:17,806 basehttp 124 "GET /addresses/ HTTP/1.1" 404 4628
INFO 2018-08-24 21:18:17,822 basehttp 124 "GET /areas/ HTTP/1.1" 200 1204
INFO 2018-08-25 14:57:30,161 basehttp 124 "GET / HTTP/1.1" 200 5218
WARNING 2018-08-25 14:57:30,391 base 152 Not Found: /favicon.ico
WARNING 2018-08-25 14:57:30,396 basehttp 124 "GET /favicon.ico HTTP/1.1" 404 4631
INFO 2018-08-25 14:57:35,310 basehttp 124 "GET /areas/ HTTP/1.1" 200 8459
INFO 2018-08-25 15:01:28,614 basehttp 124 "GET /areas/ HTTP/1.1" 200 8459
INFO 2018-08-25 15:01:30,531 basehttp 124 "GET /areas/?format=json HTTP/1.1" 200 1204
INFO 2018-08-25 16:19:22,245 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-25 16:19:24,046 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-25 16:19:25,534 basehttp 124 "GET /oauth/qq/authorization/?next=/ HTTP/1.1" 200 175
INFO 2018-08-25 16:19:35,056 basehttp 124 "OPTIONS /user/ HTTP/1.1" 200 0
WARNING 2018-08-25 16:19:35,088 basehttp 124 "GET /user/ HTTP/1.1" 401 35
INFO 2018-08-25 16:19:40,554 basehttp 124 "GET /oauth/qq/authorization/?next=/user_center_info.html HTTP/1.1" 200 196
INFO 2018-08-25 16:19:52,157 basehttp 124 "GET /oauth/qq/authorization/?next=/user_center_info.html HTTP/1.1" 200 196
INFO 2018-08-25 16:20:01,415 basehttp 124 "GET /oauth/qq/user/?code=78E8E9696E457993D0B2FD1759251293 HTTP/1.1" 200 240
INFO 2018-08-25 16:20:01,549 basehttp 124 "GET /user/ HTTP/1.1" 200 103
INFO 2018-08-25 16:20:01,567 basehttp 124 "OPTIONS /browse_histories/ HTTP/1.1" 200 0
WARNING 2018-08-25 16:20:01,589 base 152 Not Found: /browse_histories/
WARNING 2018-08-25 16:20:01,590 basehttp 124 "GET /browse_histories/ HTTP/1.1" 404 6760
INFO 2018-08-25 16:20:04,076 basehttp 124 "OPTIONS /addresses/ HTTP/1.1" 200 0
INFO 2018-08-25 16:20:04,129 basehttp 124 "GET /areas/ HTTP/1.1" 200 1204
INFO 2018-08-25 16:20:04,151 basehttp 124 "GET /addresses/ HTTP/1.1" 200 64
INFO 2018-08-25 16:20:09,726 basehttp 124 "GET /areas/120000/ HTTP/1.1" 200 74
INFO 2018-08-25 16:20:11,325 basehttp 124 "GET /areas/120100/ HTTP/1.1" 200 572
INFO 2018-08-25 16:20:19,870 basehttp 124 "POST /addresses/ HTTP/1.1" 201 220
INFO 2018-08-25 16:20:27,617 basehttp 124 "OPTIONS /addresses/1/status/ HTTP/1.1" 200 0
INFO 2018-08-25 16:20:27,642 basehttp 124 "PUT /addresses/1/status/ HTTP/1.1" 200 16
INFO 2018-08-25 16:20:34,965 basehttp 124 "GET /areas/150000/ HTTP/1.1" 200 485
INFO 2018-08-25 16:20:35,883 basehttp 124 "GET /areas/150400/ HTTP/1.1" 200 467
INFO 2018-08-25 16:20:44,334 basehttp 124 "POST /addresses/ HTTP/1.1" 201 233
INFO 2018-08-25 16:20:53,100 basehttp 124 "OPTIONS /addresses/2/ HTTP/1.1" 200 0
INFO 2018-08-25 16:20:53,142 basehttp 124 "PUT /addresses/2/ HTTP/1.1" 200 234
INFO 2018-08-25 16:20:58,241 basehttp 124 "DELETE /addresses/2/ HTTP/1.1" 204 0
INFO 2018-08-25 16:21:00,466 basehttp 124 "GET /areas/ HTTP/1.1" 200 1204
INFO 2018-08-25 16:21:00,489 basehttp 124 "GET /addresses/ HTTP/1.1" 200 281
